internal/common/runtime.go

```diff
@@ -0,0 +1,35 @@
+package common
+
+import "runtime/debug"
+
+var (
+	// Git SHA commit (only first few characters)
+	BuildCommit string
+
+	// Build date and time
+	BuildTime string
+
+	// BuildGoVersion carries Go version the binary was built with
+	BuildGoVersion string
+)
+
+func init() {
+	BuildTime = "N/A"
+	BuildCommit = "HEAD"
+
+	if bi, ok := debug.ReadBuildInfo(); ok {
+		BuildGoVersion = bi.GoVersion
+
+		for _, bs := range bi.Settings {
+			switch bs.Key {
+			case "vcs.revision":
+				if len(bs.Value) > 6 {
+					BuildCommit = bs.Value[0:6]
```

Comment:
(nitpick/drive-by) I wonder if we should use a slightly longer string here for the rev -  for the osbuild-composer repo with many commits `git log --oneline` gives me 9 chars for the git commit, this is dynamically calculated  but usually the minimum is 7. So maybe just follow git and use 9 to avoid possible collisions?